src/config.js

```diff
@@ -25,7 +25,12 @@ if (!cfg) {
 cfg.file = configPath;
 
 cfg.privateKey = async (secret) => {
-    const keystore = fs.readFileSync(path.join(homedir, '.nahmii', 'keystore', cfg.wallet.address));
+    var files = fs.readdirSync(path.join(homedir, '.nahmii', 'keystore'));
+    const matchedFile = files.filter(f => f.toLowerCase().endsWith(cfg.wallet.address.toLowerCase()))[0];
```

Comment:
A small `.replace('0x', '')` seems to be needed after `cfg.wallet.address.toLowerCase()`. Other than that I am 👍.